src/library.js

```diff
@@ -504,14 +504,14 @@ LibraryManager.library = {
 #if WASM
     var PAGE_MULTIPLE = {{{ getPageSize() }}};
     size = alignUp(size, PAGE_MULTIPLE); // round up to wasm page size
-    var old = Module['buffer'];
+    var old = buffer;
     var oldSize = old.byteLength;
```

Comment:
no need for `old` here.. just `var oldSize = buffer.byteLength;`

Reply:
Also this function seems a little confused about its return type.  Perhaps it should simply return true on sucess and false on failure?   No need to do that as part of this CL though I guess.

Reply:
Fixed the `old` stuff, thanks.

Let's think about the return type later, yeah. Offhand it looks like we check if we return a buffer, and its size - I don't remember why exactly, but might be some reason we can't just return a bool.